Makefile

```diff
@@ -52,14 +51,21 @@ IMPORT_OPTIONS := --vendor \
 		--registry-url=leader.telekube.local:5000 \
 		$(IMPORT_IMAGE_OPTIONS)
 
-TELE_BUILD_OPTIONS := --insecure \
-		--repository=$(OPS_URL) \
+TELE_BUILD_OPTIONS := --repository=$(OPS_URL) \
 		--name=$(NAME) \
 		--version=$(VERSION) \
 		--glob=**/*.yaml \
 		$(TELE_BUILD_EXTRA_OPTIONS) \
 		$(IMPORT_IMAGE_OPTIONS)
 
+TELE_BUILD_APP_OPTIONS := --insecure \
+		--hub=$(OPS_URL) \
+		--version=$(VERSION) \
+		--set registry="" \
+		--set tag=$(VERSION) \
+		--values resources/custom-values.yaml \
+		--values resources/custom-build.yaml
```

Comment:
There is no custom-build.yaml for this version.

Reply:
And overall `build-gravity-app` target was created only aiming 7.x release. This env variable is used only there. We could delete env variable and target.